plugins/tests/service.rs

```diff
@@ -74,6 +74,7 @@ fn service_with_cfg_rpc() {
 
 #[test]
 fn syntax() {
+    #[tarpc::service]
     trait Syntax {
```

Comment:
IIUC this test was supposed to test the `service` syntax. But since the attribute was missing it didn't test anything, but just produced a dead_code warning.

Reply:
Wouldn't it still be compiled, even if the compiler thinks it's dead code? What happens if a syntax error is added without `#[test]`?